views/create-service.handlebars

```diff
@@ -1,11 +1,17 @@
-<form class="new-service">
```

Comment:
changing this broke the functionality of the form, since it was triggering off that.  no worries--I'll just add an ID and make the change in my next PR